jetty-quickstart/src/main/java/org/eclipse/jetty/quickstart/QuickStartConfiguration.java

```diff
@@ -131,19 +145,23 @@ public void preConfigure(WebAppContext context) throws Exception
                 {
                     if (LOG.isDebugEnabled())
                         LOG.debug("No quickstart xml file, starting webapp {} normally", context);
-                    super.preConfigure(context);
                 }
                 break;
             }
             case QUICKSTART:
+            {
                 if (quickStartWebXml.exists())
+                {
                     quickStart(context);
+                }
                 else
-                    throw new IllegalStateException("No " + quickStartWebXml);
+                {
+                    throw new IllegalStateException("No WEB-INF/quickstart-web.xml file for " + context);
```

Comment:
Again, probably don't need the the context, as the `quickstart-web.xml` should be unique for each context.

Reply:
Context is added for those users that use more than one WebAppProvider and those use parallel deployment techniques (a feature we've supported for a while now)